addons/survey/models/survey.py

```diff
@@ -0,0 +1,1096 @@
+            ('textbox', 'Single Line Text Box'),
+            ('numerical_box', 'Numerical Value'),
+            ('datetime', 'Date and Time'),
+            ('simple_choice', 'Multiple choice: only one answer'),
+            ('multiple_choice', 'Multiple choice: multiple answers allowed'),
+            ('matrix', 'Matrix')], string='Type of Question', size=15, required=True, default='free_text')
+    matrix_subtype = fields.Selection([('simple', 'One choice per row'),
+        ('multiple', 'Multiple choices per row')], string='Matrix Type', default='simple')
+    labels_ids = fields.One2many('survey.label',
+        'question_id', string='Types of answers', oldname='answer_choice_ids', copy=True)
+    labels_ids_2 = fields.One2many('survey.label',
+        'question_id_2', string='Rows of the Matrix', copy=True)
+    # labels are used for proposed choices
+    # if question.type == simple choice | multiple choice
+    #                    -> only labels_ids is used
+    # if question.type == matrix
+    #                    -> labels_ids are the columns of the matrix
+    #                    -> labels_ids_2 are the rows of the matrix
+
+    # Display options
+    column_nb = fields.Selection([('12', '1'),
+                                   ('6', '2'),
+                                   ('4', '3'),
+                                   ('3', '4'),
+                                   ('2', '6')],
+        string='Number of columns', default='12')
+        # These options refer to col-xx-[12|6|4|3|2] classes in Bootstrap
+    display_mode = fields.Selection([('columns', 'Radio Buttons'),
+                                      ('dropdown', 'Selection Box')], default='columns')
+
+    # Comments
+    comments_allowed = fields.Boolean(string='Show Comments Field',
+        oldname="allow_comment")
+    comments_message = fields.Char(string='Comment Message', translate=True, default='If other, precise:')
+    comment_count_as_answer = fields.Boolean(string='Comment Field is an Answer Choice',
+        oldname='make_comment_field')
+
+    # Validation
+    validation_required = fields.Boolean(string='Validate entry',
+        oldname='is_validation_require', default=False)
+    validation_email = fields.Boolean(string='Input must be an email')
+    validation_length_min = fields.Integer(string='Minimum Text Length')
+    validation_length_max = fields.Integer(string='Maximum Text Length')
+    validation_min_float_value = fields.Float(string='Minimum value')
+    validation_max_float_value = fields.Float(string='Maximum value')
+    validation_min_date = fields.Datetime(string='Minimum Date')
+    validation_max_date = fields.Datetime(string='Maximum Date')
+    validation_error_msg = fields.Char(string='Error message',
+                                        oldname='validation_valid_err_msg',
+                                        translate=True, default='The answer you entered has an invalid format.')
```

Comment:
Its translatable field, why not translating default value, so when default get is called default validation_error_msg returned in user's lang